### DISCUSSION TITLE
Have you ever thought about generating a git-bug-report?

### DISCUSSION THREAD

**Discussion by ghost** [git-bug/git-bug#898] [Q&A] [answered]
Hi everyone.

My idea would be to have git-bug report - to analyze bugs. So... for analyze which bugs are most important to be made, improved, fixed... for this analysis I think of generating and using graphs such as bar graph, pie, line, timeline.

We can analyze bugs according to purpose: "ui" - a bug to improve the visual aspect of some closed or open source application. Usually people use the tag "ui" for graphical interface.

#### Image 1
![image](https://user-images.githubusercontent.com/111701513/194205100-0d4cf504-3753-435c-9c7a-a1e085b45635.png)

#### Image 2
![image](https://user-images.githubusercontent.com/111701513/194179283-3ff8f809-ddb3-4a95-b30a-d850ae0caac3.png)

**image-description:** As we can see in this image, we can see which bugs were most important according to the feature growth rate.

#### Question
*Have you ever thought about generating a bug report?*

#### why?
I was thinking to see if I could help with this idea. Nowadays there is a framework that generates a dashboard like [tremor](https://github.com/tremorlabs/tremor) - written in javascript.

**Comment by MichaelMure** [comment, marked as answer]
This is the kind of features I know less, but a lot of people (typically project manager) have opinions on, usually different ones if not contradictory.

It's doable, but I'd expect those to be made in community-created addons, in different flavors.

**Reply by ghost** [reply]
Hi MichaelMure. 

> This is the kind of features I know less, but a lot of people (typically project manager) have opinions on, usually different ones if not contradictory.

thank you for feedback.

> It's doable, but I'd expect those to be made in community-created addons, in different flavors. 

Exactly... this idea... so... my idea would be to have git-bug add-ons such as widgets, dashboards, gadgets etc.

**Reply by MichaelMure** [reply]
It wouldn't be free software if user couldn't taylor it to their needs ;-)
However, git-bug has a lot to achieve first: succeed and build a community. Github stars are not enough

**Reply by ghost** [reply]
Hi MichaelMure, thank you for feedback.
